libsolidity/parsing/Token.cpp

```diff
@@ -155,16 +155,15 @@ tuple<Token::Value, unsigned int, unsigned int> Token::fromIdentifierOrKeyword(s
 			) {
 				int n = parseSize(positionX + 1, _literal.end());
 				if (
-					0 < m && m < 256 &&
-					0 < n && n < 256 &&
+					m + n > 0 &&
```

Comment:
Yes, I am talking about the error condition that could arise when we encounter `fixed8x8888888888888888888888888888888888888888888888888888`. Please add the check for clarity even if it is also performed by `n%8==0`.